src/pages/index.tsx

```diff
@@ -11,32 +12,36 @@ const inter = Inter({
 })
 
 export default function Home() {
+  const isClient = useIsClient()
+
   return (
     <MotionConfig reducedMotion="user">
-      <AppProvider>
-        <div
-          className={`${inter.className} mx-auto prose min-h-screen w-screen max-w-6xl p-8 pb-20 antialiased sm:p-20 dark:prose-invert`}
-        >
-          <header>
-            <h1>aoe2 civ randomizer</h1>
-          </header>
+      <div
+        className={`${inter.className} mx-auto prose min-h-screen w-screen max-w-6xl p-8 pb-20 antialiased sm:p-20 dark:prose-invert`}
+      >
+        {isClient ?
+          <AppProvider>
+            <header>
+              <h1>aoe2 civ randomizer</h1>
+            </header>
 
-          <main className="flex flex-col gap-16">
-            <section>
-              <Roster />
-            </section>
+            <main className="flex flex-col gap-16">
+              <section>
+                <Roster />
+              </section>
 
-            <section className="flex flex-col items-center gap-12">
-              <CurrentCiv />
-              <Randomize />
-            </section>
+              <section className="flex flex-col items-center gap-12">
+                <CurrentCiv />
+                <Randomize />
+              </section>
 
-            <section>
-              <Configuration />
-            </section>
-          </main>
-        </div>
-      </AppProvider>
+              <section>
+                <Configuration />
+              </section>
+            </main>
+          </AppProvider>
+        : <>Loading...</>}
+      </div>
```

Comment:
We conditionally render a "Loading…" message when we're not client-side since the app actor requires `window.localStorage` to start.